src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpGoToImplementation.cs

```diff
@@ -111,7 +111,9 @@ await TestServices.Editor.SetTextAsync(
     }
 
     [IdeTheory]
-    [CombinatorialData]
+    //[CombinatorialData]
```

Comment:
Either delete this or add a comment to go back to CombinatorialData when the skip is removed.

Reply:
It's also suspicious as everything that one side would work and the other side would not, so if you just want to skip both I'm OK with that.